client/app/app.js

```diff
@@ -8,6 +8,9 @@ import Common from "./common/common";
 import PageComponents from "./components/pages";
 import AppMaterial from "./app.material";
 import AppTranslate from "./app.translate";
+import AppXEditableConfig from "./app.xeditable";
+import "angular-xeditable";
```

Comment:
I've got no better solution for this, so I would just use this style. Similar to the mapintegration.